projects/igniteui-angular/src/lib/select/select.component.html

```diff
@@ -0,0 +1,33 @@
+<igx-input-group #inputGroup class="input-group" [disabled]="disabled" (click)="toggle()">
+    <ng-container ngProjectAs="[igxLabel]">
+        <ng-content select="[igxLabel]"></ng-content>
+    </ng-container>
+    <ng-container ngProjectAs="igx-prefix">
+        <ng-content select="igx-prefix"></ng-content>
+    </ng-container>
+
+    <input #input class="input" type="text" igxInput [igxSelectItemNavigation]="this"
+        readonly= "true"
+        [placeholder]="this.placeholder"
+        [(ngModel)]="this.selectionValue"
+        [attr.aria-labelledby]="this.label ? this.label.id : ''"
+        [attr.role]="'combobox'"
+        [attr.aria-expanded]="ariaExpanded"
+        [attr.aria-haspopup]="ariaHasPopUp"
```

Comment:
`aria-haspopup="listbox"` is not going to change, so it can be a static sttribute (not bound). Same applies for the listbox role of the container.